### DISCUSSION TITLE
asking for the labelled data

### DISCUSSION THREAD

**Discussion by manasgithub005** [ultralytics/ultralytics#8195] [Q&A]
For further finetuning model

**Comment by pderrenger** [comment]
Hi there! 👋

For fine-tuning your YOLOv8 model, you'll need a labeled dataset. If you're looking for publicly available datasets, COCO and VOC are great places to start, and they can be automatically downloaded when you use them for the first time with YOLOv8.

If you have your own custom dataset, make sure it's labeled in the format YOLOv8 expects. Here's a quick example of how to start training with a custom dataset:

```python
from ultralytics import YOLO

# Load a pretrained model
model = YOLO('yolov8n.pt')

# Fine-tune the model on your custom dataset
results = model.train(data='your_dataset.yaml', epochs=50)
```

Make sure `your_dataset.yaml` points to your labeled data and is formatted correctly. Happy fine-tuning! 😊

If you need further assistance, feel free to ask.
